docs/theming-flatpak.rst

```diff
@@ -0,0 +1,56 @@
+Theming Flatpak
+===============
+
+Flatpak apps default to Adwaita as a theme, as they cannot use the system theme. This happens because all flatpaks are sandboxed and do not have the ability to read data files or libraries in ``/usr`` (where system themes are typically located). This is done so that the format is portable across a wide range of distributions in a reproducible environment, but has the downside of not being able to pick up themes. The solution to this was to package GTK themes as Flatpaks, as relying upon the host to have the correct version for every flatpak defeats the portability benefits it provides. These themes are provided as extensions to the Gnome runtime.
+
+Installing themes
+-----------------
+
+The current themes are packaged in the flathub repository which you can add (if it's not already added) by running::
+
+  $ flatpak remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
+
+To see a list of currently packaged themes you can run::
+
+  $ flatpak remote-ls flathub | grep org.gtk.Gtk3theme
```

Comment:
As mentioned in my other comment we now have `flatpak search gtk3theme`

Reply:
If you don't mind me asking, since which version is this the case? Because I have Flatpak 0.10.0 and it says that the command 'search' is unknown. And given that this is the version included by default on Linux Mint 18.3 (the first to include builtin Flatpak support btw), I'll probably have to keep both command sets (because many people might be averse to updating it via a PPA), and just specify from which version onwards the 'search' one can be used.
P.S. Idle thought of mine: I just thought what would happen if Flatpak itself could be updated via... Flatpak. Probably impossible, but interesting that in order for Flatpak itself to be updated on Ubuntu-based distros, a PPA is needed.

Reply:
Thanks for the info. Given that the PPA route is yours (the developer's) stance on the matter, I took the liberty to add that having older versions of Flatpak is discouraged by the developers.